test/network-request.spec.ts

```diff
@@ -45,6 +45,35 @@ it('should work for fetch requests', async ({page, server}) => {
   expect(requests[0].frame()).toBe(page.mainFrame());
 });
 
+it('should work for a redirect', async ({page, server}) => {
+  server.setRedirect('/foo.html', '/empty.html');
+  let requests = [];
+  page.on('request', request => requests.push(request));
+  requests = requests.filter(request => !request.url().includes('favicon'));
+  await page.goto(server.PREFIX + '/foo.html');
+
+  expect(requests.length).toBe(2);
+  expect(requests[0].url()).toBe(server.PREFIX + '/foo.html');
+  expect(requests[1].url()).toBe(server.PREFIX + '/empty.html');
+});
+
+it('should work for a redirect and interception', async ({page, server}) => {
```

Comment:
Could you please update this to "should not work for a redirect and interception" and expect a single request instead?

Reply:
done!